modules/transformer-batch/src/test/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/batch/good/CrossBatchDeduplicationSpec.scala

```diff
@@ -227,11 +184,13 @@ object CrossBatchDeduplicationSpec {
  * @see https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html
  */
 class CrossBatchDeduplicationSpec extends Specification with ShredJobSpec {
-  args(skipAll = true)
-
+  val storage = CrossBatchDeduplicationSpec.Storage.prepareLocalTable()
```

Comment:
Oh, so this test has never been really run on CI, because DynamoDB instance was needed? And now we run it on CI, but with in memory replacement? 

I guess the old spec was useful if you had local DynamoDB instance launched and you wanted to run this spec manually against actual AWS stuff. It's gone now... but will anyone miss that? Just wondering how useful that was in practice.  

We could keep the old spec and add new 'in memory' next to it, but that would probably only make too much fuss? Anyway, until we have testcontainers or sth like that here, in memory approach looks like a very good solution :)

Reply:
Yep, using DynamoDB mock is definitely better way but I didn't want to spend too much time on it tbh. I think even this change is improvement since this test isn't run at all previously. We can improve it later on 🙂